src/javascript/Localizations.js

```diff
@@ -15,14 +15,15 @@ const Localizations = {
     'ru', // Russian
     'zh_CN', // Simplified Chinese
     'zh_TW' // Traditional Chinese
```

Comment:
comma missing, extra comma in next line

Reply:
Updated this in the same branch - File 'Update Localizations.js'.